projects/kit/components/input-files/input-files.template.html

```diff
@@ -14,14 +14,19 @@
         (tuiDroppableDragOverChange)="onDragOver($event)"
         (mousedown.prevent.silent)="(0)"
     >
-        <a tuiLink>
-            <ng-container *polymorpheusOutlet="computedLink$ | async as text">
+        <ng-container *ngIf="content; else defaultContent">
+            <div *polymorpheusOutlet="content as valueContent">{{ valueContent }}</div>
```

Comment:
`as primitiveContent` is more accurate, because it represents template for string content.